src/components/PeopleTable/TableHead/TableHead.tsx

```diff
@@ -0,0 +1,54 @@
+import { useSearchParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { SearchLink } from '../../PeopleFilters/SearchLink';
+import { SearchParams } from '../../../utils/searchHelper';
+
+const SORTFIELDS = ['Name', 'Sex', 'Born', 'Died'];
+
+export const TableHeader = () => {
+  const [searchParams] = useSearchParams();
+  const currentSort = searchParams.get('sort');
+  const currentOrder = searchParams.get('order');
+
+  const iconClass = (sortBy: string): string => {
```

Comment:
The function name must begin with a verb
```suggestion
  const getIconClass = (sortBy: string): string => {
```